<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1035</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_pregunta">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>441</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Pregunta: ¿Como estas bb?</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_opc1">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Opción 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_opc2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>140</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Opción 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_opc3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>140</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Opción 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_opc5">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>190</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Opción 5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_opc4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>190</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Opción 4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>Hemos encontrado unas recomendaciones para usted</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_recomendaciones">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>421</width>
      <height>281</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>dsnfjdshjf s jfhsjfk sahjasfb as fashfjsa fashfasb f asujfbas fasfhsjf saf aufgabfan fasjfba f as fahfasb  fs ahfewyvf ds fegbfsdn f</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>PRESUPUESTO</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_presupuesto">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>100</y>
      <width>511</width>
      <height>451</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Imagen</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Descripción</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Cantidad</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Precio Unidad</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Total</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btn_imprimir">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>620</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Imprimir Presupuesto</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_salir">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>650</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_txt_total">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>580</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TOTAL: $</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_total">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>580</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>750</string>
    </property>
   </widget>
   <zorder>table_presupuesto</zorder>
   <zorder>lbl_pregunta</zorder>
   <zorder>btn_opc1</zorder>
   <zorder>btn_opc2</zorder>
   <zorder>btn_opc3</zorder>
   <zorder>btn_opc5</zorder>
   <zorder>btn_opc4</zorder>
   <zorder>label_2</zorder>
   <zorder>lbl_recomendaciones</zorder>
   <zorder>label_4</zorder>
   <zorder>btn_imprimir</zorder>
   <zorder>btn_salir</zorder>
   <zorder>lbl_txt_total</zorder>
   <zorder>lbl_total</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
